### DISCUSSION TITLE
传输文件的困惑，0——0 大佬的帮忙回答下!

### DISCUSSION THREAD

**Discussion by liaobadadi** [AlistGo/alist#1391] [Q&A]
情况：部署了ALIST到腾讯云，但是腾讯云上是每个月就300G流量，而且带宽狠小慢。
看了说明书，还是不太懂，
proxy：网页端和直链走不走中转
webdav_proxy：webdav文件下载走不走中转
webdav_direct：webdav是否直接由本机中转

1 .PROXY 关闭，webdav_direct：是不是 网页和直链不走腾讯云流量？ 

2.我上传文件的时候，有提示后台处理中，那是不是只要是上传，不管你如何设置，都是要走腾讯云流量?

3.网页端看视频的时候，是不是走腾讯云流量。

感谢大佬回答！

**Comment by cxw620** [comment]
看文档, https://alist-doc.nn.ci/docs/driver/base#proxy%E4%BB%A3%E7%90%86%E7%9B%B8%E5%85%B3

**Reply by liaobadadi** [reply]
感谢！找到方法了
